08LessonEight/homework/exercise-06.html

```diff
@@ -3,8 +3,42 @@
 
 <head>
     <title>Exercise 06</title>
+    <link type="text/css" rel="stylesheet" href="css/exercise-06-style.css" />
 </head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+<body>
+    <main>
+        <section class="mainframe clearfix">
+            <div class="infobox clearfix">

```

Comment:
Again, you are using `clearfix` to the elements that are floating (same with elements on line 21 and 31)
`clear-fix` it's applied to the **parent** when all child elements are floating - not to childs